lib/eex/lib/eex/compiler.ex

```diff
@@ -195,16 +197,16 @@ defmodule EEx.Compiler do
 
   # Look middle expressions that immediately follow a start_expr
 
-  defp look_ahead_middle([{:text, _, _, text} | rest], start, contents) do
+  defp look_ahead_middle([{:text, text, _meta} | rest], start, contents) do
     if only_spaces?(text) do
       look_ahead_middle(rest, start, contents ++ text)
     else
       nil
     end
   end
 
-  defp look_ahead_middle([{:middle_expr, line, _column, _, chars} | rest], _start, contents) do
-    {rest, line, contents ++ chars}
+  defp look_ahead_middle([{:middle_expr, chars, meta} | rest], _start, contents) do
```

Comment:
Here is the bug:

```suggestion
  defp look_ahead_middle([{:middle_expr, _, chars, meta} | rest], _start, contents) do
```

:)

Reply:
ohh man, I looked at it so many times but still couldn't find it 🙈 

Thanks ❤️